src/plone/app/multilingual/tests/test_lif.py

```diff
@@ -14,11 +14,12 @@
 from zope.event import notify
 from zope.interface import alsoProvides
 from zope.interface import noLongerProvides
-from zope.lifecycleevent import ObjectModifiedEvent
+from zope.lifecycleevent import ObjectModifiedEvent, modified
```

Comment:
You may want to apply isort/black here. I am not sure if this was done already and how many changes this would imply (if many, better on own PR). But at least do one import per line:

```suggestion
from zope.lifecycleevent import modified
from zope.lifecycleevent import ObjectModifiedEvent
```

Reply:
@jensens Oh hmm, I knew this was wrong while typing it but I was being lazy and figured it would get corrected on save, or during precommit, or at least flagged by a lint CI job to remind me! I'll check why none of those happened.